way_to_home/notification/models.py

```diff
@@ -1 +1,68 @@
 """This module implements class that represents the notification entity."""
+
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+from way.models import Way
+
+
+WEEK = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+
+
+class Notification(AbstractModel):
+    """Model for Notification entity."""
+
+    way = models.ForeignKey(Way, on_delete=models.CASCADE)
+    start_time = models.DateTimeField()
+    end_time = models.DateTimeField()
+    week_day = models.PositiveSmallIntegerField()
+    time = models.TimeField()
+
+    def __str__(self):
+        """Method that returns route instance as string."""
+
+        return f'notification at: {WEEK[self.week_day]} {str(self.time)}'
+
+    def to_dict(self):
+        """Method that returns dict with object's attributes."""
+
+        return {
+            'id': self.id,
+            'start_time': self.start_time,
+            'end_time': self.end_time,
+            'week_day': self.week_day,
+            'time': self.time,
+            'way': self.way.id
```

Comment:
```suggestion
            'way': self.way.id
```
U can just use `way_id`. Check it in other models also, please :)

Reply:
Done, in User, Route and Notification models